tests/device_tests/bitcoin/test_multisig.py

```diff
@@ -233,24 +231,31 @@ def test_missing_pubkey(client):
         assert exc.value.message.endswith("Pubkey not found in multisig script")
 
 
-def test_attack_change_input(client):
+@pytest.mark.multisig
+def test_attack_change_input(client: TrezorClientDebugLink):
     """
     In Phases 1 and 2 the attacker replaces a non-multisig input
     `input_real` with a multisig input `input_fake`, which allows the
     attacker to provide a 1-of-2 multisig change address. When `input_real`
     is provided in the signing phase, an error must occur.
+
+    NOTE: previously this test was making sure that MatchChecker's check_input()
+    function catches the change in the account type. Recently this check was replaced
+    by verifying the scriptPubKey.
+    The goal of the test was to replicate the "Malicious change in mixed transactions" attack in
+    https://blog.trezor.io/details-of-firmware-updates-for-trezor-one-version-1-9-0-and-trezor-model-t-version-2-3-0-46deb141fc09
```

Comment:
I suggest:

The goal of this test is to replicate the "Malicious change in mixed transactions" attack in https://blog.trezor.io/details-of-firmware-updates-for-trezor-one-version-1-9-0-and-trezor-model-t-version-2-3-0-46deb141fc09
Previously this test was making sure that MatchChecker's check_input() function catches the change in the account type. After scriptPubKey verification was introduced, the attack is caught earlier by the scriptPubKey mismatch.

Reply:
Looks good, I will update it